pkg/ddl/ingest/disk_root.go

```diff
@@ -155,3 +193,20 @@ func (d *diskRootImpl) StartupCheck() error {
 func RiskOfDiskFull(available, capacity uint64) bool {
 	return float64(available) < (1-capacityThreshold)*float64(capacity)
 }
+
+// CheckIngestLeakageForTest is only used in test.
+func CheckIngestLeakageForTest(exitCode int) {
```

Comment:
can move to `testutil` pkg

Reply:
There is an import cycle. Tests in `ingest` depends on `testutil`, and `testutil` depends on `ingest`.

Reply:
we can make the pkg of `TestMain` `ingest_test`